bounties_api/user/views.py

```diff
@@ -58,6 +59,15 @@ def get(self, request, public_address=''):
             {'nonce': user.nonce, 'has_signed_up': bool(user.email) and bool(user.name)})
 
 
+class DismissSignup(APIView):
+    permission_classes = [AuthenticationPermission, UserIDMatches]
+
+    def get(self, request, public_address=''):
+        user = User.objects.get(public_address=public_address.lower())
+        user.dismissed_signup_prompt = datetime.utcnow()
```

Comment:
This is not being saved, is that intentional?

Reply:
yep - I do need to save it :) haha I was rushing and good catch